elastic/kibana
# [Fleet] fix for install latest GA bug

## Summary

Closes https://github.com/elastic/kibana/issues/149056

Fixes issue where latest GA version couldn't be installed if there is a newer prerelease version in the registry. The fix works regardless of the beta switch being on or off.

Test steps:
1. Open Kibana, Go to Integrations
2. Enable beta integrations (to verify a beta version is available)
3. Select Elastic Defend 
4. Install latest GA version (v8.6.1)
5. The install should finish successfully

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/90178898/213148936-1fd1c01e-5295-4eca-9f2e-928d07d9da77.png">
<img width="1776" alt="image" src="https://user-images.githubusercontent.com/90178898/213149081-1ed6f395-692c-4512-b4d1-cbce704b22fc.png">


### Checklist
